src/main/java/robot/subsystems/drivetrain/commands/PurePursue.java

```diff
@@ -219,6 +220,20 @@ private double distanceLookahead() {
                 Math.sqrt(Math.pow(a, 2) + Math.pow(b, 2));
     }
 
+    /**
+     * Takes the current output, and gradually raises it to the target velocity, while making sure it doesn't change
```

Comment:
Change the documentation into imperative form. Also, a few tweaks.
```suggestion
     * Take the current output and gradually raise it to the target velocity, while making sure it doesn't change
```